README.md

```diff
@@ -1,295 +1,314 @@
-        'items',
-        security_packet,
-        consumer_secret,
-        items_request
-)
+## Installation
+###  **Installation via RubyGems**
+Using RubyGems is the recommended way to install the Learnosity SDK for Ruby in production. The easiest way is to run this from your project folder:
 
-puts init.generate
-```
+    gem install learnosity_sdk
 
-This will return a string of signed options suitable for initialisation of
-the API.
+### **Alternative method 1: download the zip file**
+Download the latest version of the SDK as a self-contained ZIP file from the [GitHub Releases](https://github.com/Learnosity/learnosity-sdk-ruby/releases) page. The distribution ZIP file contains all the necessary dependencies. 
 
-```html
-<html>
-    <head>
-    </head>
+Note: after installation, run this command in the SDK root folder:
 
-    <body>
-	    <script src="//items.learnosity.com/"></script>
+    bundle install
 
-	    <script>
-	    var itemsApp = LearnosityItems.init(INSERT OPTIONS HERE);
-	    </script>
+### **Alternative 2: development install from a git clone**
+To install from the terminal, run this command:
 
-    </body>
-</html>
-```
+    git clone git@github.com:Learnosity/learnosity-sdk-Ruby.git
 
-### Data API
+Note: after installation, run this command in the SDK root folder:
 
-When the `service` parameter is `data`, the `action` parameter is mandatory.
-Moreover, `Init#generate`'s `encode` parameter is ignored, and a native Ruby Hash
-with the signed options is unconditionally returned, for use with your favourite
-HTTP library (note that, regardless of the `action` parameter, you should always
-send `POST` requests).
+    bundle install
 
-```ruby
-require 'net/http'
-require "learnosity/sdk/request/init"
+Note that these manual installation methods are for development and testing only.
+For production use, you should install the SDK using the Pip package manager for Ruby, as described above.
 
+[(Back to top)](#table-of-contents)
 
-security_packet = {
-	# XXX: This is a Learnosity Demos consumer; replace it with your own consumer key
-	'consumer_key'   => 'yis0TYCu7U9V4o7M',
-	'domain'         => 'localhost'
-}
-# XXX: The consumer secret should be in a properly secured credential store, and *NEVER* checked in in revision control
-consumer_secret = '74c5fd430cf1242a527f6223aebd42d30464be22'
-data_request = { 'limit' => 50 }
+## Quick start guide
+Let's take a look at a simple example of the SDK in action. In this example, we'll load an assessment into the browser.
 
-init = Learnosity::Sdk::Request::Init.new(
-        'data',
-        security_packet,
-        consumer_secret,
-        data_request,
-	'get'
-)
+### **Start up your web server and view the standalone assessment example**
+To start up your Ruby web server, first find the following folder location under the SDK. Change directory ('cd') to this location on the command line.
 
-request = init.generate
+    learnosity-sdk-ruby/examples/lrn-sdk-rails/
```

Comment:
Let's make this a valid command (also, it is customary to show commands from the root of the project [though, arguably, one would have to get into that directory first, after cloning or extraction])

```suggestion
    cd examples/lrn-sdk-rails/
```